extensions/interactions/DragAndDropSortInput/directives/drag_and_drop_sort_input_interaction_directive.html

```diff
@@ -59,29 +47,40 @@
     position: relative;
   }
 
-  .drag-and-drop-sort-form-container img {
+  .drag-and-drop-sort-container img {
     max-height: 200px;
   }
+
+  .conversation-skin-interaction-container {
+    /* This CSS class is essential for auto-scrolling in Drag and Drop Sort
```

Comment:
This still doesn't talk about why this modification is essential. It just repeatedly says it is essential.

Explain the last sentence more: _how_ and _why_ will the drag-and-drop interaction break if this class is removed?

Reply:
```
This CSS class is essential for auto-scrolling in Drag and Drop Sort
interaction to work correctly. We need to set specific CSS rules just for this
interaction so that auto-scrolling works correctly. These CSS rules are
intentionally defined here so that they do not affect any other interaction.
If this class is removed, the auto-scrolling in Drag and drop interaction will break
as this container will not be scrollable in case draggable items are large in number.
So, it has to be set as overflow:scroll with definite max-height.
```

 Is it better now? After that, I'll make a final commit.

Reply:
Yep it's better! I did a bit of rewording; consider using this:

```
This CSS class is essential for auto-scrolling in the Drag and Drop Sort interaction to work correctly. They are intentionally defined here so that they do not affect any other interaction.

If this class is removed, the auto-scrolling in Drag and drop interaction will break, as this container will not be scrollable when there are a lot of draggable items (see issue #5280). So, it has to be set as overflow:scroll with a definite max-height.
```

Reply:
Done.